core/src/main/java/com/novoda/noplayer/PlayerView.java

```diff
@@ -15,4 +19,6 @@
     void showSubtitles();
 
     void hideSubtitles();
+
+    void setSubtitleCue(List<Cue> cues);
```

Comment:
It would be better if we could avoid exposing the Cues here as it is an exoPlayer class. Maybe we could create a Cue class of our own ?